extension/observer/k8sobserver/config.go

```diff
@@ -34,12 +34,14 @@ type Config struct {
 	// it will only discover node endpoints whose `metadata.name` matches the provided node name. If `true` and
 	// Node isn't specified, it will discover all available node endpoints. `false` by default.
 	ObserveNodes bool `mapstructure:"observe_nodes"`
+	// ObserveServices determines whether to report observer service and port endpoints. `false` by default.
+	ObserveServices bool `mapstructure:"observe_services"`
```

Comment:
Not needed w/ these changes but I wonder if moving to an inner `observe` map is preferable as we'd add more types.